Tools/ToolsCommon/Data/Viewports/ViewportTemplates.json

```diff
@@ -63,6 +63,16 @@
         "y": 443,
         "width": 170,
         "height": 170
+      },
```

Comment:
adding these viewports to the base file (ViewportTemplates.json) asserts that these patches will succeed on every version of DCS since 41371 or whatever is the first version we can actually patch.   Is that intentional?   I am ok with it, just making sure we are all aware of what it means.   The alternative would be to move all the affected aircraft into the versioned files, which is probably why you asked about that in the first place.    Maybe we just do this, and if anyone comes back later saying they failed to patch an ancient version of DCS, then you have to fix it :)

Reply:
These changes have now been moved into a versioned file and i have bought the baked KA50 templates up to the latest file too as discussed. As such the hud exports will only be available in DCS 2.7.6.13436 or later.